.github/workflows/rerun-compare-chromium-versions.yml

```diff
@@ -0,0 +1,47 @@
+name: Rerun Compare Chromium versions
+
+on:
+  pull_request:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+.x'
+    paths:
+      - package.json
+    types:
+      - closed
+
+jobs:
+  rerun-compare-chromium-versions:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_SHA: ${{ github.event.pull_request.head.sha }}
+      TARGET_SHA: ${{ github.event.pull_request.base.sha }}
+    steps:
+      - run: |
+          shopt -s inherit_errexit
```

Comment:
This should call a script in the repo so it can be run independently of github. This is similar to having code in the CI repo itself that runs as part of the build. Even scripts that only run on CI should be in brave-core